frontend/src/components/toast/toast-duplicate-input-button.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Center from "react-center";
+import ToastButton from "./toast-button";
+import Style from "style-it";
+
+/*Expects an array and duplicates the "children".  Set maxItems to limit the number of duplicates.  
+To pass in value(s) that only need to be shown once refer to toast-show-hide-input. */
+class ToastDuplicateInputButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.duplicateInput = this.duplicateInput.bind(this);
+  }
+
+  getClasses() {
+    let classes = "";
```

Comment:
change this to classes = [] and push hideBtn to the array. then return classes.join(' '). this is what i typically do. it makes it easier to add more classes later if you need them